modules/apps/web-experience/layout/layout-admin-web/src/main/resources/META-INF/resources/configuration/icon/copy_applications.jsp

```diff
@@ -32,7 +32,7 @@ PortletURL redirectURL = layoutsAdminDisplayContext.getRedirectURL();
 	<portlet:param name="mvcPath" value="/view.jsp" />
 </portlet:actionURL>
 
-<aui:form action='<%= HttpUtil.addParameter(copyApplicationsURL, "refererPlid", plid) %>' name="fm">
+<aui:form action='<%= HttpUtil.addParameter(copyApplicationsURL, "refererPlid", plid) %>' name="fm_copyApplications">
```

Comment:
@brianchandotcom, why do we standardize on `fm` for all forms. I'd expect similar ID collisions elsewhere, which are not immediately apparent. This bug, specifically, isn't obvious unless you were auditing the validation on this page.

I named these forms after their `actionURL` `name`.